### PR TITLE
docs: add angular version compatibility table

### PR DESCRIPTION
**Pull request by hoebbelsB**
# Description

added a table which displays which rxa versions support which angular version

### CODE REVIEW

**Review by edbzn — APPROVED** [review]

**Review by edbzn — APPROVED** [review]

File: libs/template/README.md [hunk 1]
```diff
@@ -107,8 +107,16 @@ about what changes are leading to re-renderings.
...
+### Angular
+|               | @rx-angular/template |
+|---------------|----------------------|
+| Angular > 13  | `> 1.0.0-beta.29`    |
+| Angular > 12  | `<= 1.0.0-beta.29`   |
```
Comment (BioPhoton) [inline libs/template/README.md:116]:
```suggestion
|   Angular      |   RxJS                                                     | @rx-angular/template |
|---------------|-------------------------------------------|--------------------------|
|  > 13.1.x       | 6.5.x/6.6.x/7.4.x or later minor version  | > 1.0.0-beta.29             |
| < 12.1.4        | 6.5.x/6.6.x                                              | <= 1.0.0-beta.29          |
```

File: libs/state/README.md [hunk 2]
```diff
@@ -135,8 +135,17 @@ nx migrate @rx-angular/state
...
+### Angular
+|               | @rx-angular/state   |
+|---------------|---------------------|
+| Angular > 13  | `> 1.4.6`           |
+| Angular > 12  | `<= 1.4.6`          |
```
Comment (BioPhoton) [inline libs/state/README.md:144]:
```suggestion
|   Angular      |   RxJS                                                     | @rx-angular/state  |
|---------------|-------------------------------------------|----------------------|
|  > 13.1.x       | 6.5.x/6.6.x/7.4.x or later minor version  | > 1.4.6                    |
| < 12.1.4        | 6.5.x/6.6.x                                              | <= 1.4.6                 |
```

File: libs/cdk/README.md [hunk 3]
```diff
@@ -48,8 +48,16 @@ ng update @rx-angular/cdk
...
+### Angular
+|               | @rx-angular/cdk     | 
+|---------------|---------------------|
+| Angular > 13  | `> 1.0.0-alpha.10`  |
+| Angular > 12  | `<= 1.0.0-alpha.10` |
```
Comment (BioPhoton) [inline libs/cdk/README.md:57]:
```suggestion
|   Angular      |   RxJS                                                     | @rx-angular/cdk    |
|---------------|-------------------------------------------|----------------------|
|  > 13.1.x       | 6.5.x/6.6.x/7.4.x or later minor version  | > 1.0.0-alpha.10     |
| < 12.1.4        | 6.5.x/6.6.x                                              | <= 1.0.0-alpha.10   |
```

File: README.md [hunk 4]
```diff
@@ -27,8 +27,16 @@ Find details in the linked readme files below for installation and setup instruc
...
-| --------- | --------- | --------- | --------- | --------- |
-| IE11, Edge| last version| last version| last 2 versions| last 2 versions
+### Angular
+|               | @rx-angular/state   | @rx-angular/template | @rx-angular/cdk     | 
+|---------------|---------------------|----------------------|---------------------|
+| Angular > 13  | `> 1.4.6`           | `> 1.0.0-beta.29`    | `> 1.0.0-alpha.10`  |
+| Angular > 12  | `<= 1.4.6`          | `<= 1.0.0-beta.29`   | `<= 1.0.0-alpha.10` |
```
Comment (BioPhoton) [inline README.md:36]:
```suggestion
|   Angular      |   RxJS                                                     | @rx-angular/state   | @rx-angular/template | @rx-angular/cdk     | 
|---------------|-------------------------------------------|----------------------|----------------------|----------------------|
|  > 13.1.x       | 6.5.x/6.6.x/7.4.x or later minor version  | > 1.4.6                    | > 1.0.0-beta.29       | > 1.0.0-alpha.10     |
| < 12.1.4        | 6.5.x/6.6.x                                              | <= 1.4.6                  | <= 1.0.0-beta.29    | <= 1.0.0-alpha.10  |
```

**Comment by edbzn** [comment]
@hoebbelsB @BioPhoton What is wrong or missing in this PR?

**Comment by hoebbelsB** [comment]
@edbzn we are uncertain if it's actually true what we are writing here :D

**Pull request merged**